mne/utils/docs.py

```diff
@@ -144,6 +144,18 @@
 docdict['reject_by_annotation_raw'] = docdict['reject_by_annotation_all'] + """
     Has no effect if ``inst`` is not a :class:`mne.io.Raw` object.
 """
+docdict['annot_ch_names'] = """
+ch_names : array-like | None
+    Array-like of lists of channel names associated with the annotations.
+    Empty entries or entries with None are assumed to be associated with all
+    channels (i.e., with the time slice itself). For example, to create two
+    annotations, associating the second with two channels could be done
+    by using ``ch_names`` like::
+
+        Annotations(onset=[0, 3, 10], duration=[1, 0.25, 0.5],
+                    description=['Start', 'BAD_flux', 'BAD_noise'],
+                    ch_names=[[], ['MEG0111', 'MEG2563']])
```

Comment:
You describe creating two annotations, but the example shows three.

Also, would it make sense to also support channel indexes as an alternative to channel names? That way, ranges could be more easily specified. But I guess there's always the `pick_*` route, which maybe should be mentioned in another example if we don't want to support it directly.

Reply:
I think working with channel names will generally be easier for people, especially once picking and reordering gets involved.

This does remind me, though, that I need to fix `rename_channels` to git into `annotations.ch_names` now, though.

Reply:
Agreed. Indices are more error prone

Reply:
@larsoner but the example is still confusing/wrong, could you clarify?